smarts/core/smarts.py

```diff
@@ -1127,6 +1127,11 @@ def should_reset(self):
         """If the simulation requires a reset."""
         return self._reset_required
 
+    @property
+    def resetting(self):
```

Comment:
Nit since it is evident:
```suggestion
    def resetting(self) -> bool:
```